### PR TITLE
vdk-core: [Hot Fix] Stop throwing exceptions if config.ini not present

### PR DESCRIPTION
**Pull request by doks5**
This change disables the checks for config.ini files when creating instances of JobConfig. This is needed because if someone tries to execute a local vdk run help (`vdk run --help`) depending on their distribution's plugins, this may result in errors.

### CODE REVIEW

**Review by tozka — APPROVED** [review]

File: projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py
Comment (tozka) [inline projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py:65]: I feel you have commented out more than needed. Shouldn't only the raise VdkConfigurationError be commented. 

Also you may add log.info statement.

File: projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py
Comment (doks5) [reply projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py:47]: Hi, @tozka 
In the old times, this was the way we were doing it, https://github.com/vmware/versatile-data-kit/commit/06eb4a51a1eacc5f633d8b6ebdcef011fb682512

File: projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py
Comment (doks5) [reply projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py]: You're correct. I refactored it to only log a message and not throw an error. Thank you for the review

**Review by mivanov1988 — APPROVED** [review]

File: projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py
Comment (doks5) [reply projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py]: Also, according to the documentation, it just returns an empty dataset, https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read

Comment (tozka) [reply projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py]: Oh, ok.

 There's no testing done in the section. Before merging make sure to test it. As trivial as it may seem the change, it's aways possible to miss something. We may even add an automated test for that as we do not want to fail if config.ini is missing. 

It's approved so merge it when you feel it's ready.

Comment (doks5) [reply projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py]: Agree, I tested it locally, so it seems to be working as expected.

**Pull request merged**
